client/app/components/NoTaggedObjectsFound.jsx

```diff
@@ -6,9 +6,9 @@ import { BigMessage } from './BigMessage';
 function NoTaggedObjectsFound({ objectType, tags }) {
   return (
     <BigMessage icon="fa-tags">
-      No {objectType} found tagged with
+      No {objectType} found tagged with&nbsp;
       {Array.from(tags).map(tag => (
-        <span className="label label-tag" key={tag}>
+        <span className="label label-tag label--inline" key={tag}>
```

Comment:
I think we can now use [TagsControl](https://github.com/getredash/redash/blob/master/client/app/components/tags-control/TagsControl.jsx) here. If it's okay - I can implement it tomorrow (in this PR) (cc @arikfr)

Reply:
Will it also fix the alignment issue?

Reply:
Don't know :-) I just decided that it's more consistent to use existing component for displaying tags instead of rendering custom HTML